src/mkdocstrings/plugin.py

```diff
@@ -87,6 +88,8 @@ class MkdocstringsPlugin(BasePlugin):
               options:
                 selection_opt: 2
     ```
+
+    - **`enabled`**: Whether or not the plugin is enabled. Default is `True`. If false, all hooks exit early.
```

Comment:
```suggestion
    - **`enabled`**: Whether to enable the plugin. Default is true. If false, *mkdocstrings* will not collect or render anything.
```

Also please move the line up, after `default_handler` for example.